Modules/Loadable/Markups/MRML/vtkMRMLMarkupsDisplayNode.h

```diff
@@ -518,5 +529,9 @@ class  VTK_SLICER_MARKUPS_MODULE_MRML_EXPORT vtkMRMLMarkupsDisplayNode : public
   bool RotationHandleVisibility;
   bool ScaleHandleVisibility;
   double InteractionHandleScale;
+
+  bool RotationHandleAxes[4];
```

Comment:
We need to add "Visible" or "Enabled" to the name.

Also, "handle" and "axis" feels somewhat redundant. Since the free translation handle is not really an axis, probably "handle" would be more appropriate.

So, the name could be something like `RotationHandleVisibility` or `RotationHandleComponentVisibility` (or `...Enabled` instead of `...Visibility`).

Same for scale and translation.